scripts/src/javatests/org/oppia/android/scripts/license/MavenDependenciesRetrieverTest.kt

```diff
@@ -34,6 +34,23 @@ class MavenDependenciesRetrieverTest {
   private val DEP_WITH_DIRECT_LINK_ONLY_LICENSE = "com.google.firebase:firebase-analytics:17.5.0"
   private val DEP_WITH_INVALID_LINKS = "io.fabric.sdk.android:fabric:1.4.7"
 
+  private val DATA_BINDING_DEP_WITH_THIRD_PARTY_PREFIX =
+    "//third_party:androidx_databinding_databinding-adapters"
+  private val PROTO_DEP_WITH_THIRD_PARTY_PREFIX =
+    "//third_party:com_google_protobuf_protobuf-lite"
+  private val GLIDE_DEP_WITH_THIRD_PARTY_PREFIX =
+    "//third_party:com_github_bumptech_glide_annotations"
+  private val FIREBASE_DEP_WITH_THIRD_PARTY_PREFIX =
+    "//third_party:com_google_firebase_firebase-analytics"
+  private val IO_FABRIC_DEP_WITH_THIRD_PARTY_PREFIX =
+    "//third_party:io_fabric_sdk_android_fabric"
+
+  private val DATA_BINDING_DEP = "androidx_databinding_databinding_adapters"
+  private val PROTO_DEP = "com_google_protobuf_protobuf_lite"
+  private val GLIDE_DEP = "com_github_bumptech_glide_annotations"
+  private val FIREBASE_DEP = "com_google_firebase_firebase_analytics"
+  private val IO_FABRIC_DEP = "io_fabric_sdk_android_fabric"
```

Comment:
```suggestion
  private val DATA_BINDING_DEP = "androidx_databinding_databinding_adapters"
  private val PROTO_DEP = "com_google_protobuf_protobuf_lite"
  private val GLIDE_DEP = "com_github_bumptech_glide_annotations"
  private val FIREBASE_DEP = "com_google_firebase_firebase_analytics"
  private val IO_FABRIC_DEP = "io_fabric_sdk_android_fabric"

  private val DATA_BINDING_DEP_WITH_THIRD_PARTY_PREFIX =
    "//third_party:$DATA_BINDING_DEP"
  private val PROTO_DEP_WITH_THIRD_PARTY_PREFIX =
    "//third_party:$PROTO_DEP"
  private val GLIDE_DEP_WITH_THIRD_PARTY_PREFIX =
    "//third_party:$GLIDE_DEP"
  private val FIREBASE_DEP_WITH_THIRD_PARTY_PREFIX =
    "//third_party:$FIREBASE_DEP"
  private val IO_FABRIC_DEP_WITH_THIRD_PARTY_PREFIX =
    "//third_party:$IO_FABRIC_DEP"
```

To ensure that the constants are correctly associated. Ditto for the other test suite.

Also, if any of these can be wrapped to one line, please update.

Reply:
Actually, the ones listed above are a bit different, they can't have a hyphen '-' in them (only underscores) but the ones with third_party prefix can have a hyphen, e.g- 
```
private val DATA_BINDING_DEP = "androidx_databinding_databinding_adapters"
private val DATA_BINDING_DEP_WITH_THIRD_PARTY_PREFIX =
    "//third_party:androidx_databinding_databinding-adapters"
```

Reply:
Ah, got it--that was subtle. Thanks for clarifying.